index.html

```diff
@@ -90,6 +86,27 @@ <h1>Nabla Core API sample app</h1>
                 <textarea id="patient-context"></textarea>
             </div>
 
+            <div class="filters">
+                <label for="note-sections">Section to customize below:</label>
+                <select id="note-sections" onChange="onSectionChange()"></select>
+            </div>
+
+            <div id="section-customization-fields">
+                <div class="filters">
+                    <label for="style-select">Style:</label>
+                    <select id="style-select" onChange="onCustomizationChange()">
+                        <option value="">(No preference)</option>
```

Comment:
the actual value you can pass and that is the default is `AUTO`
```suggestion
                        <option value="AUTO">Auto</option>
```